src/clients/c++/experimental_api_v2/library/base_utils.h

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
So there are some common classes between HTTP and GRPC... but they are not really utils are they? Can they just be in a common.h/.cc?

Reply:
Fixed.